Hearthstone Deck Tracker/Controls/Overlay/BattlegroundsMinions.xaml.cs

```diff
@@ -72,13 +78,17 @@ private void Update(int tier, IEnumerable<Race> availableRaces)
 				for(var i = 0; i < 6; i++)
 					_tierIcons[i].SetFaded(false);
 				Groups.Clear();
+				UnavailableTypes.UnavailableTypesVisibility = System.Windows.Visibility.Collapsed;
```

Comment:
Is there any scenario where all minion types are available currently? We might be able to get rid of all the visibility changes/logic.